src/main/java/com/precourse/NumberGenerator.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2019 Gyeonglok Kim
+ * All rights reserved.
+ */
+
+package com.precourse;
+
+import java.util.ArrayList;
+
+/**
+ * 클래스 이름: NumberGenerator.java
+ * 기능1) 0~9 사이의 서로 다른 3개의 숫자를 만드는 기능(정답)
+ *
+ * @author rok
+ * @version 1.0.0
+ * <p>
+ * 날짜: 2019.12.04 (수)
+ */
+public class NumberGenerator {
+    private static final int ANSWER_LENGTH = 3;        // 정답의 길이 3으로 고정
+    private static final int RANGE_OF_NUMBER = 9;
+    private static ArrayList<Integer> answerList = new ArrayList<>();
+
+    /**
+     * 정답으로 설정할 숫자 리스트 생성하는 메서드
+     * @return
+     */
+    public static ArrayList<Integer> numberGenerator() {
```

Comment:
```suggestion
    public static List<Integer> numberGenerator() {
```

[이펙티브자바 52 : 객체를 참조할 때는 그 인터페이스를 사용하라](https://ktko.tistory.com/entry/Effective-Java-52-%EA%B0%9D%EC%B2%B4%EB%A5%BC-%EC%B0%B8%EC%A1%B0%ED%95%A0-%EB%95%8C%EB%8A%94-%EA%B7%B8-%EC%9D%B8%ED%84%B0%ED%8E%98%EC%9D%B4%EC%8A%A4%EB%A5%BC-%EC%82%AC%EC%9A%A9%ED%95%98%EB%9D%BC)

객체를 참조할 때에는 인터페이스를 구현하는 클래스보다 인터페이스를 사용하는 것이 프로그램의 유연성에 있어 더 좋습니다.